rotkehlchen/balances/manual.py

```diff
@@ -32,24 +37,37 @@ class ManuallyTrackedBalanceWithValue(NamedTuple):
     location: Location
     tags: list[str] | None
     balance_type: BalanceType
+    asset_is_missing: bool = False
 
     def serialize(self) -> dict[str, Any]:
         result = self._asdict()  # pylint: disable=no-member
         del result['value']
+        if self.asset_is_missing is False:
```

Comment:
Is this extra logic needed? It's to save space in the response right? But you also then add an extra if at every serialization